Backpack/src/main/java/net/skyscanner/backpack/badge/BpkBadge.kt

```diff
@@ -34,15 +34,15 @@ open class BpkBadge @JvmOverloads constructor(
     /**
      * Style for badges with positive messages
      */
-    Success(1, R.color.bpkGreen500, R.color.bpkGray700),
+    Success(1, R.color.bpkMonteverde, R.color.bpkGray700),
```

Comment:
HorizontalNavTests.screenshotHorizontalNav_alternate need to have the background colour updated.

Reply:
done